instance-billing-flavor-check.spec

```diff
@@ -0,0 +1,53 @@
+#
+# spec file for package instance-billing-type-check
+#
+# Copyright (c) 2022 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+Summary:        Cloud Billing Flavour Check
+Name:           python-instance-billing-type-check
+Version:        0.0.1
+Release:        0
+License:        GPL3.0-only
+Group:          Productivity/Networking/Web/Servers
+URL:            https://github.com/SUSE-Enceladus/instance-billing-flavor-check
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-roo
+Requires:       python3
+Requires:       python3-lxml
+Requires:       python3-requests
+BuildRequires:  python3-setuptools
+BuildRequires:  python-rpm-macros
+
+%python_subpackages
+
+%description
+Check if instance is PAYG or BYOS
+
+%prep
+%setup -q
+
+%build
+# Build Python 3 version
+%python_build
+
+%install
+%python_install
+mkdir -p %{buildroot}%{python3_sitelib}/instance_flavor_check
```

Comment:
The `mkdir` just creates an empty directory. The actual package directory is `instance_billing_flavor_check`, and `setup.py` creates the directory. However the package name as per `setup.py` is `instance-flavor-check` which means the egg info dir is `instance_flavor_check-0.0.1-py3.10.egg-info`. That's not a problem per se but I'd use the same name for consistency.